<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
    <meta content="" name="description"/>
    <meta content="" name="author"/>
    <!-- For iPhone -->
    <meta name="msapplication-TileColor" content="#4076e0">
    <!-- Chrome, Firefox OS and Opera -->
    <meta name="theme-color" content="#4076e0">
    <!-- Windows Phone -->
    <meta name="msapplication-navbutton-color" content="#4076e0">
    <title>Virtual Labs</title>
    <link rel="shortcut icon" href="./assets/images/favicon.ico" />
    <link rel="stylesheet" href="./assets/css/bootstrap.min.css" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/github-markdown-css/4.0.0/github-markdown.min.css" integrity="sha512-Oy18vBnbSJkXTndr2n6lDMO5NN31UljR8e/ICzVPrGpSud4Gkckb8yUpqhKuUNoE+o9gAb4O/rAxxw1ojyUVzg==" crossorigin="anonymous" />
    <link
      href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Raleway&display=swap"
      rel="stylesheet"
    />
    <link
      rel="stylesheet"
      href="./assets/fonts/font-awesome-4.7.0/css/font-awesome.min.css"
    />
    <link rel="stylesheet" href="./assets/css/github-markdown.min.css" />
    <link rel="stylesheet" href="./assets/css/vlabs-style.css" />
    
    <script src="./assets/js/jquery-3.4.1.slim.min.js"></script>
    <script
      src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.5.4/dist/umd/popper.min.js"
      integrity="sha384-q2kxQ16AaE6UbzuKqyBE9/u/KzioAlnx2maXQHiDX9d4/zp8Ok3f+M7DPm+Ib6IU"
      crossorigin="anonymous"
    ></script>
    <script
      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/js/bootstrap.min.js"
      integrity="sha384-pQQkAEnwaBkjpqZ8RU1fF1AKtTcHJwFl3pblpTlHXybJjHpMYo79HY3hIi4NKxyj"
      crossorigin="anonymous"
    ></script>
    <style>
      h1 {
      font-size: 1.8rem;
      }
      h2 {
      font-size: 1.4rem;
      }
      h3 {
      font-size: 1.2rem;
      }

      h2, h3 {
      font-weight: 800;
      }
    </style>
  </head>
  <body class="p-0 container-fluid vlabs-page d-flex flex-column justify-content-between">
    <header class="vlabs-header sticky-top bg-white">
      <nav class="p-0 navbar navbar-light d-flex align-items-stretch">
        
        <button class="navbar-toggler px-4"
        	    type="button" onclick="toggle()">
          <span class="navbar-toggler-icon"></span>
        </button>
    
        <div class="d-flex justify-content-center">        
          <a href="http://vlab.co.in" class="navbar-brand">
            <img src="./assets/images/vlabs-color-small-moe.jpg" alt="vlabs logo">
          </a>
        </div>
        
        <div id="headerNavbar" class="border-top flex-grow-1 d-flex align-items-center">
          <ul class="navbar-nav ml-auto text-center d-flex flex-md-row">
            <li class="nav-item px-2 m-1">
              <a href="http://vlab.co.in">HOME</a>
            </li>
            <li class="nav-item px-2 m-1">
              <a href="http://vlab.co.in/participating-institutes">PARTNERS</a>
            </li>
            <li class="nav-item px-2 m-1">
              <a href="http://vlab.co.in/contact-us">CONTACT</a>
            </li>
          </ul>
        </div>
      </nav>
    </header>
    <div class="container-fluid flex-fill d-flex flex-column vlabs-page-main">
      <div class="row flex-grow-1 px-4 d-flex flex-nowrap">
        <div class="nav flex-column sidebar flex-shrink-0" id="menu">
            <div>
              <div class="d-flex">
                <a href="index.html" class="p-2 ">
                  Aim
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="theory.html" class="p-2  current-item ">
                  Theory
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="pretest.html" class="p-2 ">
                  Pretest
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="procedure.html" class="p-2 ">
                  Procedure
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="simulation.html" class="p-2 ">
                  Simulation
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="posttest.html" class="p-2 ">
                  Posttest
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="references.html" class="p-2 ">
                  References
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="feedback.html" class="p-2 ">
                  Feedback
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
        </div>        <div class="vlabs-page-content px-4 pb-4 flex-grow-1 markdown-body">
          <div class="text-center">
            <h2 id="querying-ncbi-database-in-r">Querying NCBI database in R</h2>
        
          </div>
            <p><strong>Introduction</strong></p>
        <p>There has been an increasing demand in biological sciences for deep understanding of biological data that were generated by the human genome project and other sequencing projects.  Bioinformatics, an interdisciplinary field provides tools for computational analysis and interpretation of biological data especially in modern medicine and biology. Using complex computer software programs, DNA and protein sequence data of specific organisms were retrieved and sorted for analysis and predictions by molecular biologists. Pharmaceutical companies were also relying on constant input from bioinformatics experts for acquisition and analysis of biological data for large scale analysis. Biological data from genome projects put forward the need of algorithm software programs for storing the biological data in a usable format. Many databases exist through internet for retrieving information such as DNA sequence of species, genetic mutations and polymorphisms and so on. Understanding biological databases using search engines provide repositories of gene data for identifying and analyzing genetic data for biomedical research. </p>
        <p>&nbsp;</p>
        <p><strong>National Center for Biotechnology Information (NCBI)</strong></p>
        <p>NCBI (<a href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov/</a>) was developed by National Library of Medicine (NLM) at the National Institutes of Health in 1988 for the purpose of developing information systems for molecular biology. It is a comprehensive website that has databases relevant for biotechnology and biomedicine. It provides retrieval sources for the data in GenBank, a comprehensive database with publicly available nucleotide sequences of various species and other biological data through the NCBI website. GenBank has participation with the European Molecular Biology Lab (EMBL): Nucleotide Sequence Database (EMBL-Bank), and the DNA Data Bank of Japan (DDBJ). Examples of NCBI data resources include Entrez, Pubmed, BLAST, COBALT, the Map Viewer, Gene Expression Omnibus, Molecular Modeling Database and so on, which can be accessed through NCBI homepage. </p>
        <p>&nbsp;</p>
        <p>The stipulated roles of NCBI multidisciplinary research includes</p>
        <ol>
        <li>   Largest repository for biological research data by creating public databases for storage, retrieval, analysis of data related to molecular biology, genetics and biochemistry.</li>
        <li>   Integrating scientific and medical data with computer-based information processing systems which emphasizes research in computational biology related to structure and function of biological molecules.</li>
        <li>   Designing algorithms for biological data analysis and cross-referencing mechanisms for genome analysis.</li>
        <li>   Developing software tools for genome sequence analysis and for disseminating biomedical information.</li>
        </ol>
        <p>&nbsp;</p>
        <p><strong>Entrez Databases</strong>
        Major goal in databases development was to provide a user friendly access to stored biological data. A wide number of biological retrieval systems exist for genome analysis. Among these, the most popular one is Entrez, an integrated search engine, developed and maintained by NCBI for retrieval of biological data. It allow users for text based searches for nucleotide and protein sequence data, structural information, abstracts, citations, and taxonomic data related to specific query. It integrates information by cross-referencing with NCBI databases on the basis of preexisting and logical relationships between the individual query sequences.</p>
        <p> &nbsp;</p>
        <p> <strong>NCBI database in biological sciences</strong>
        In NCBI Sequence Database, each sequence has a separate record and a unique identifier which can be used to refer to that sequence record. Unique identifier is recorded as accession, consisting of numbers and letters. For example, accessions for the DNA sequences of the Dengue Virus types are DEN-1 (NC_001477), DEN-2 (NC_001474), DEN-3(NC_001475) and DEN-4 (NC_002640) respectively. For example, to retrieve DNA sequence of DEN-2 virus, potential virus to cause dengue hemorrhagic fever, in NCBI website (<a href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov/</a>) type “NC_001474 in search box and press the “Search” button. </p>
        <p>&nbsp;</p>
        <center><img src="images/1.png" title="" /></center>
        
        
        <center><img src="images/2.png" title="" /></center>
        
         <center>Fig.1. Querying sequence data for Dengue virus, type 2</center>
        
        <p>&nbsp;</p>
        <p>The result page retrives “PubMed” data containing abstracts from scientific papers, Nucleotide database contains DNA and RNA sequence analysis, the “Protein” data contains for retrieving protein sequence data, and so on. User can look into different features of the given query. Instead of providing accession number, users can type query word such as insulin, keratin etc and can search in depth about the query sequence. 
        To retrieve FASTA format sequence of DEN-2 Dengue virus genome, click Send” at the top right of the NC_001477 sequence record webpage and choose “File” in FASTA and save the file in “ My documents folder of user’s personal computer as den2.fasta (Fig.2). </p>
        <p> &nbsp;</p>
        <center><img src="images/3.png" title="" /></center>
        &nbsp;
         
        <center>Fig.2. Retrieving FASTA file of query sequence</center>
        &nbsp;
        
        
        <p>For detailed information see:
        <a href="http://vlab.amrita.edu/index.php?sub=3&amp;brch=273&amp;sim=1437&amp;cnt=1">http://vlab.amrita.edu/index.php?sub=3&amp;brch=273&amp;sim=1437&amp;cnt=1</a> </p>
        <p>&nbsp;</p>
        <p><strong>Querying genome sequence data using SeqinR</strong>
        Apart from retrieving genome sequence from NCBI website, users can retrieve sequence data from NCBI directly from R. With the standard R installation add-ons need to be loaded for running R programming function. SeqinR package helps in retrieving sequences from a DNA sequence database, for carrying out simple DNA sequence analyses.</p>
        <p>Bioconductor set of R packages (<a href="http://www.bioconductor.org">www.bioconductor.org</a>) has packages for R programming for analyzing biological data sets such as microarray data. 
        SeqinR package (pbil.univ-lyon1.fr/software/seqinr/home.php?lang=eng) works for retrieving DNA sequences and protein sequence databases, DNA and protein sequences analyses.</p>
        
        </div>
      </div>
    </div>
    <footer class="text-light pt-2 vlabs-footer d-flex flex-column">
      <div class="row px-5">
        <div class="col d-flex flex-column">
          <span class="font-weight-bold vlabs-footer-sect-name pb-2 mb-3">Community Links</span>
          <a href="http://www.sakshat.ac.in/" class="text-light">Sakshat Portal</a>
          <a href="http://outreach.vlabs.ac.in/" class="text-light">Outreach Portal</a>
          <a href="http://vlab.co.in/faq" class="text-light">FAQ: Virtual Labs</a>
        </div>
        <div class="col d-flex flex-column">
          <span class="font-weight-bold vlabs-footer-sect-name pb-2 mb-3">Contact Us</span>
          <span> Phone: General Information: 011-26582050 </span>
          <span> Email: support@vlabs.ac.in </span>
        </div>
        <div class="col d-flex flex-column">
          <span class="font-weight-bold vlabs-footer-sect-name pb-2 mb-3">Follow Us</span>
          <div class="social-links">
    	<a class="p-2 mt-1 mr-2 d-inline-flex justify-content-center align-items-center" style="background: #55acee;" href="https://twitter.com/TheVirtualLabs">
    	  <i class="fa fa-twitter"></i>
    	</a>
    	<a class="p-2 mt-1 mr-2 d-inline-flex justify-content-center align-items-center" style="background: #3b5998;" href="https://www.facebook.com/Virtual-Labs-IIT-Delhi-301510159983871/">
    	  <i class="fa fa-facebook"></i>
    	</a>
    	<a class="p-2 mt-1 mr-2 d-inline-flex justify-content-center align-items-center" style="background: #e52d27;" href="https://www.youtube.com/watch?v=asxRaOgk6a0">
    	  <i class="fa fa-youtube"></i>
    	</a>
    	<a class="p-2 mt-1 mr-2 d-inline-flex justify-content-center align-items-center" style="background: #2867B2;" href="https://in.linkedin.com/in/virtual-labs-008ba9136">
    	  <i class="fa fa-linkedin"></i>
    	</a>
          </div>
        </div>
      </div>
      <div class="m-0 py-2 text-center" style='font-family: "Open Sans", sans-serif; background: #212121;'>
        <a class="text-primary font-weight-bold" href="https://www.gnu.org/licenses/agpl-3.0.en.html"> AGPL 3.0 </a>
        &nbsp;&amp;&nbsp;
        <a class="text-primary font-weight-bold" href="https://creativecommons.org/licenses/by-nc-sa/4.0/"> Creative Commons (CC BY-NC-SA 4.0) </a>
      </div>
    </footer>

      <script src="./assets/js/toggleSidebar.js"></script>
      
  </body>
</html>
